src/content/learn/state-as-a-snapshot.md

```diff
@@ -61,43 +61,41 @@ label, textarea { margin-bottom: 10px; display: block; }
+1. O React chama sua função novamente.
+2. Sua função retorna uma nova foto instantânea do JSX.
+3. O React, então, atualiza a tela para corresponder à foto instantânea que você retornou.
 
 <IllustrationBlock sequential>
-    <Illustration caption="React executing the function" src="/images/docs/illustrations/i_render1.png" />
-    <Illustration caption="Calculating the snapshot" src="/images/docs/illustrations/i_render2.png" />
-    <Illustration caption="Updating the DOM tree" src="/images/docs/illustrations/i_render3.png" />
+    <Illustration caption="React executando a função" src="/images/docs/illustrations/i_render1.png" />
+    <Illustration caption="Calculando a foto instantânea" src="/images/docs/illustrations/i_render2.png" />
+    <Illustration caption="Atualizando a árvore DOM" src="/images/docs/illustrations/i_render3.png" />
 </IllustrationBlock>
 
-As a component's memory, state is not like a regular variable that disappears after your function returns. State actually "lives" in React itself--as if on a shelf!--outside of your function. When React calls your component, it gives you a snapshot of the state for that particular render. Your component returns a snapshot of the UI with a fresh set of props and event handlers in its JSX, all calculated **using the state values from that render!**
+Assim como a memória de um componente, o *state* não é como uma variável regular que desaparece após a função retornar. O *state* realmente "vive" no React, fora da sua função. Quando o React chama seu componente, ele lhe dá uma foto instantânea do *state* para aquela renderização específica. Seu componente retorna uma foto instantânea da interface do usuário com um novo conjunto de *props* e manipuladores de eventos em seu JSX, todos calculados **usando os valores do *state* daquela renderização!**
 
 <IllustrationBlock sequential>
-  <Illustration caption="You tell React to update the state" src="/images/docs/illustrations/i_state-snapshot1.png" />
-  <Illustration caption="React updates the state value" src="/images/docs/illustrations/i_state-snapshot2.png" />
-  <Illustration caption="React passes a snapshot of the state value into the component" src="/images/docs/illustrations/i_state-snapshot3.png" />
+  <Illustration caption="Você diz ao React para atualizar o state" src="/images/docs/illustrations/i_state-snapshot1.png" />
+  <Illustration caption="React atualiza o valor do state" src="/images/docs/illustrations/i_state-snapshot2.png" />
+  <Illustration caption="React passa uma foto instantânea do valor do state para o componente" src="/images/docs/illustrations/i_state-snapshot3.png" />
 </IllustrationBlock>
 
-Here's a little experiment to show you how this works. In this example, you might expect that clicking the "+3" button would increment the counter three times because it calls `setNumber(number + 1)` three times.
-
-See what happens when you click the "+3" button:
```

Comment:
Me parece que essa linha não foi incluída na tradução.

- See what happens when you click the "+3" button:
- Veja o que acontece quando você clica no botão "+3":